### DISCUSSION TITLE
Why doesn't Dark Reader work automatically?

### DISCUSSION THREAD

**Discussion by jparrie** [darkreader/darkreader#7193] [Q&A]
Set to "Auto" or "On", i expect the app to work on every site I visit, But it doesn't unless I have already manually clicked the "disabled for current website" box. What am I missing or is this the only way to get it to work?

**Comment by Gusted** [comment]
Hi!

I think you've _accidentally_ disabled the "Enabled by default" option in the settings.

Regards,
Gusted

**Comment by jparrie** [comment]
Well, I thought that was the answer, but I still land on websites where the app doesn't work unless I manually start the app.

**Reply by bershanskiy** [reply]
Which websites? Since you might have messed up your settings, would you mind sharing your settings to rule out this hypothesis? To do that:
 - open popup and click on "Dev tools", on popup which will appear click "Preview new design"
 - open popup again and click "Settings" > "Manage settings" > "Export settings"
 - uploaded file which will be downloaded
